scripts/shared/cleanup.sh

```diff
@@ -24,6 +24,7 @@ function stop_local_registry {
 ### Main ###
 
 run_all_clusters delete_cluster
+rm -rf ${DAPPER_OUTPUT}/*
```

Comment:
Can we add a check to make sure $DAPPER_OUTPUT is not empty? 
```suggestion
if [ "${DAPPER_OUTPUT}" != "" ]; then
    rm -rf ${DAPPER_OUTPUT}/*
fi    
```
I was imagining running "cleanup.sh" from bash, by mistake :D

Reply:
Makes sense, thanks!